README.md

```diff
@@ -93,22 +93,27 @@ on information how to set this up.
 
 ```bash
 # source global ros
-$ source /opt/ros/<your_ros_version>/setup.bash
+source /opt/ros/<your_ros_version>/setup.bash
+
 # create a catkin workspace
-$ mkdir -p catkin_ws/src && cd catkin_ws
-$ catkin_make
-$ cd src
+mkdir -p catkin_ws/src && cd catkin_ws
+
 # clone the driver
-$ git clone <this_repository_url>
+git clone https://github.com/UniversalRobots/Universal_Robots_ROS_Driver.git src/Universal_Robots_ROS_Driver
+
 # clone fork of the description to use the calibration feature
-$ git clone -b calibration_devel https://github.com/fmauch/universal_robot.git
+git clone -b calibration_devel https://github.com/fmauch/universal_robot.git src/fmauch_universal_robot
+
 # install dependencies
-$ rosdep install --from-path . -y --ignore-src
-# build the driver
-$ cd ..
-$ catkin_make
-# source the workspace
-$ source devel/setup.bash
+sudo apt update -qq
+rosdep update
+rosdep install --from-path src --ignore-src -y
+
+# build the workspace
+catkin_make
+
+# activate the workspace (ie: source it)
+source devel/setup.bash
```

Comment:
I personally prefer having a `$` in front of each command, which makes it visually easier to understand when there are multi-line commands (at least in my opinion). If this is nothing usually being done, I'm fine to remove them.

Reply:
I understand why you say this, and I feel it does make line-continuation easier to spot, but I've answered enough questions on ROS Answers about "why does bash not execute my command which I just copy-pasted from a tutorial?" to not do this any more.


It's about 50/50 whether this is caused by incomplete copy-pastes, or because they copied line nrs and/or `$`-signs into the shell.

Inexperienced users will not realise they should not include the `$`, while experienced users will already know how to interpret these lists of commands. For those it will just make copy-pasting sets of commands more difficult.

As there is no way to toggle the `$`s, I've switched to removing them from snippets where the intent is for users to actually execute the commands verbatim (compared to, say, in examples which need the user to update the commands before they can execute them).

Reply:
If you'd prefer I could add the `$`s back though.

Reply:
That does make sense to me, yes. But then I'd prefer making this consistently through all code snippets in this repo's READMEs.

Reply:
> then I'd prefer making this consistently through all code snippets in this repo's READMEs

sure. But that seems a little out-of-scope for this PR, which was specifically about the build instructions in the main readme.

Reply:
Then I would vote for leaving in the `$`s for this PR and we tackle them in a separate commit.